assets/app/features/upgrade/controllers/upgrade-administration-repositories-checks.controller.js

```diff
@@ -67,20 +68,24 @@
                         });
                         // Iterate over the checks to determine the validity of the step
                         vm.repoChecks.valid = Object.keys(vm.repoChecks.checks).every(function(k) {
+                            if (vm.repoChecks.checks[k].status === false) {
+                                repoErrors.push(vm.repoChecks.checks[k].label)
+                            }
                             return vm.repoChecks.checks[k].status === true
                         });
 
                         // if all the repos are ok, complete the step
                         if (vm.repoChecks.valid) {
                             upgradeStepsFactory.setCurrentStepCompleted();
+                        } else {
+                            // TODO(itxaka): This is a tmp thing, title and style should be clarified
+                            vm.error = { title: 'Some repositories are not ready', body: repoErrors.join(', ') };
```

Comment:
After some discussion around the implementation of nodes repo checks, we decided that showing error popup for failed repos doesn't bring much benefit as the same information will be displayed in the main step page anyway (red marks).